scripts/masterCron.ts

```diff
@@ -562,7 +568,7 @@ async function run() {
 
   var commitAndCloseJob = new CronJob(
     // 0 0 10 * * 5 = 10am UTC on Fridays.
-    `0 0 ${COMMIT_START} * * 5`,
+    `0 ${NETWORK_CONGESTION_BUFFER * 3} ${COMMIT_START} * * 5`,
```

Comment:
Can we just specify the exact minutes it will execute? Not sure multiplying the `NETWORK_CONGESTION_BUFFER` helps with readability